gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -766,6 +802,40 @@ public void testGet() {
   }
 
   @Test
+  public void testLookupEventualConsistency() {
+    DatastoreRpcFactory rpcFactoryMock = EasyMock.createStrictMock(DatastoreRpcFactory.class);
+    DatastoreRpc rpcMock = EasyMock.createStrictMock(DatastoreRpc.class);
+    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(DatastoreOptions.class)))
```

Comment:
Also here maybe we can first create the `DatastoreOptions` and use that object instead of `EasyMock.anyObject(DatastoreOptions.class)`